mlrun/api/main.py

```diff
@@ -347,6 +356,39 @@ def _cleanup_runtimes():
         close_session(db_session)
 
 
+def _push_run_notifications(db: mlrun.api.db.base.DBInterface, db_session):
+
+    # Import here to avoid circular import
+    import mlrun.api.api.utils
+
+    # When pushing notifications, push notifications only for runs that entered a terminal state
+    # since the last time we pushed notifications.
+    # On the first time we push notifications, we'll push notifications for all runs that are in a terminal state.
+    global _last_notification_push_time
```

Comment:
I think the comment for what it is being used for should also be written at the definition of the global variable